chia/full_node/coin_store.py

```diff
@@ -432,6 +432,7 @@ async def get_coin_states_by_ids(
         include_spent_coins: bool,
         coin_ids: Set[bytes32],
         min_height: uint32 = uint32(0),
+        max_height: uint32 = uint32(uint32.MAXIMUM_EXCLUSIVE - 1),
```

Comment:
I think it would be good to add this after the `*`

Reply:
Changed